README.md

```diff
@@ -1,18 +1,29 @@
 # Twilio Voice React Native Reference
 
-## What is this Project?
+This project consists of a backend server and a React Native app demonstrating best practices for developing a Twilio Programmable Voice application.
 
-The Twilio Voice React Native Reference App is an example implementation of the Twilio Voice React Native SDK and serves to inspire developers who want to leverage the power of Twilio Programmable Voice in their React Native applications.
+## Features
 
-* See the Twilio Voice React Native SDK [here](https://github.com/twilio/twilio-voice-react-native).
+* Dialpad for making outgoing calls to a phone number
+* Active call screen to interact with and control ongoing calls
 
-## Before you use this Project
+### Planned Features
 
-You should have the React Native development environment set up and have a firm understanding of Twilio Programmable Voice.
+* Call to a Twilio Client
```

Comment:
This should already work. Are we unable to establish client-to-client calls, or do we mean from app to app?

Reply:
I meant app to app. I'll update.